start.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Please, add proper instructions, on the README.md, on how to use the `start.sh` executable at launch, and how to set the right variables, so that each 'profiled' service can start.

Reply:
@jorgeantonio21 Btw, I'm not very happy with the `start.sh`. 
DO you have a suggestion on how to get rid of it? Many node operators will not like running a bash script (and rightly so). But, at the same time, I didn't see another way to add some logic (e.g. make sure user does not select 2 chat completions backends at the same time. This will be relevant when we have more than 1 backend for chat completions)

Reply:
So, I figured out the most optimal way:
- the validation can happen implicitly on the `container_name`. Just tested, and there cannot be 2 containers with the same name, so if user tries to spin 2 containers for chat completions, docker compose will throw that error.
- No bash required

THis is solved in https://github.com/atoma-network/atoma-node/pull/205